packages/adapters/ethers/src/client.ts

```diff
@@ -14,8 +14,8 @@ import {
   StorageUtil,
   getPreferredAccountType
 } from '@reown/appkit-controllers'
+import { ProviderController } from '@reown/appkit-controllers'
```

Comment:
Moving ProviderUtil (actually a controller) to ProviderController. Previously the ProviderUtil was located in `/appkit-utils` package where calling ProviderUtil from different packages causing some referencing issues. That's why `useAppKitProvider` (who uses `useSnapshot` from `valtio`) was not working properly with Valtio v2.

Reply:
had this in my refactor, looks ok